### ISSUE TITLE
Custom channel enter greeting: Whenever someones enters the chat room (if requested by the user)

### ISSUE THREAD

**Issue by swedishwb** [hardlydifficult/HardlyBot#35]
The message "Swedish Says Hello" (custom message defined by the regular)...

 Would save me alot of time not having to say hello  :P 

User > !setgreeting "Swedish says hi"
Bot > KK

User enters channel
Bot > Swedish says hi.

**Comment by theBuildr45** [comment]
have a colum in sql named gretting default "Welcome"

**Comment by HardlyDifficult** [comment]
Agree.  I think it should go in the twitch_followers table.  Maybe that table is renamed.  It should hold info about the user+channel such as greeting and isFollowing.  i.e. you should be able to create a greeting without first following... technically.
